data/json/items/gun/223.json

```
@@ -292,13 +196,89 @@
       }
     ]
   },
+  {
+    "//": "First group change from 'NATO Assault Rifle'. Groups all 'H&K416' related rifles together, alongside the SCAR-L due to its similar properties. Used M27 due to it being the most popular of the four in the US.",
```

It doesn't really matter what the previous state was, just what it is now.
```suggestion
    "//": "Groups all 'H&K416' related rifles together, alongside the SCAR-L due to its similar properties. Used M27 due to it being the most popular of the four in the US.",
```